com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -304,11 +308,6 @@ private void Init(bool server)
                 MessageHandler = new InternalMessageHandler(this);
             }
 
-            NetworkSceneManager.RegisteredSceneNames.Clear();
```

Comment:
I assume we no longer need this because, even if we call Init() multiple times (e.g. call StartServer again after stopping) we are creating a new NetworkSceneManager, and the old one will be out of scope and be freed

Still, is there value in, if there already is a NetworkSceneManager, that we call perhaps a cleanup function on it that would do things like the code that was removed?  And then ditto for the others here (CustomMessagingManager, BufferManager, etc)

Reply:
This way we do it currently fully cleans it up though. (Lists etc are actually thrown out instead of their Count being set to 0 and their underlying array still being present).